.chloggen/introduce_sha2Hasher.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: attributesprocessor
```

Comment:
It's also used in the resource processor

Reply:
added thanks!